index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="header">
+    <div class="up"><img src="icons/Dark.png"></div>
+    <div class="logo">My Logo</div>
+    <div class="union"><img src="icons/Union.png"></div>
+  </div> 
+  <div class="headline">
+    <div class="text">Hey There. <br> I'm Alex.</div>
```

Comment:
Kindly update this element to use a `<h1>` tag. This is for semantic reasons, as this is the main title of the page🙏